example/hands-on/create-hands-on-app.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+########################
+# rn-spoilerをテンプレートとして、学習コンテンツの内容を適用したアプリの作成
+# 第一引数：アプリ名(Optional. デフォルト値:HandsOnApp)
+########################
+
+set -euo pipefail
+APP_NAME=${1:-HandsOnApp}
+
+echo "Start creating $APP_NAME."
+
+# 実行ファイルのディレクトリを取得
+EXECUTABLE_SCRIPT_DIR=$(cd $(dirname $0); pwd)
+# パッチファイルが格納されているディレクトリを取得
+PATCHES_DIR=$EXECUTABLE_SCRIPT_DIR/patches
+
+cd $EXECUTABLE_SCRIPT_DIR
+# rn-spoilerの親ディレクトリに移動して、そこに新規アプリを作成する
+cd ../../
+RN_SPOILER_DIR=$(pwd)
+# アプリを構築するディレクトリを作成
+mkdir -p generated
+cd generated
+WORK_DIR=$(pwd)
+cd $WORK_DIR
```

Comment:
`WORK_DIR`は先に定義しておいた方が良い気がする。
こんな感じとか？

```suggestion
WORK_DIR="${ROOT_DIR}/generated"
mkdir -p "${WORK_DIR}"
cd "$WORK_DIR"
```

Reply:
先に定義するようにしましたー